spec/command/list_spec.rb

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2013 whiteleaf. All rights reserved.
+#
+
+require "time"
+require "timecop"
+require "commandline"
+
+describe Command::List do
+  let(:novels) { Database.instance.get_object.values }
+  let(:list) do
+    Command::List.new.tap do |this|
+      this.instance_variable_set(:@options, options)
+    end
+  end
+  let(:options) { {} }
+  let(:max) { novels.size }
+
+  define :be_size_of do |expected|
+    match do |actual|
+      actual.size == expected
+    end
+  end
+
+  describe "#output_list" do
+    def expect_receive_output_lines_with_be_size_of(size)
+      expect(list).to receive(:output_lines).with(be_size_of size)
```

Comment:
__[rubocop]__
* Style/NestedParenthesizedCalls: Add parentheses to nested method call `be_size_of size`.